src/constants/actions.js

```diff
@@ -0,0 +1,13 @@
+// App
+export const SET_MOVIES_SORT_BY = "APP/SET_MOVIES_SORT_BY";
+
+// Movies
+export const REQUEST_MOVIES_SUCCESS = "MOVIES/REQUEST_MOVIES_SUCCESS";
```

Comment:
you may add MOVIES/REQUEST_MOVIES_INIT to indicate that loading process is started and show spinner on that to improve user experience.

Reply:
i will add a comment in the code for that and will do it later 😄